### PR TITLE
Update adding_a_vmware_vcenter_provider.md for customer reported issue.

### PR DESCRIPTION
**Pull request by dehawkins512**
Doc update to address downstream customer reported issue  12961.

### CODE REVIEW

**Comment by dehawkins512** [comment]
@Fryguy FYI, please merge.

**Comment by chessbyte** [comment]
@agrare once we have your review approval, @Fryguy or I will merge this PR.

File: managing_providers/_topics/adding_a_vmware_vcenter_provider.md
Comment (agrare) [inline managing_providers/_topics/adding_a_vmware_vcenter_provider.md:28]: > **Note**: This is only available for VMware ESXi hosts.

This is redundant when we're in the "Add a vCenter" section since ESXi are the only type of host that are available on vCenter.

Comment (dehawkins512) [reply managing_providers/_topics/adding_a_vmware_vcenter_provider.md]: removed the **Note**: This is only available for VMware ESXi hosts.

**Review by agrare — APPROVED** [review]

**Pull request merged**
